spec/pundit_spec.rb

```diff
@@ -37,18 +37,10 @@
       expect(Pundit.authorize(user, [:project, :admin, comment], :update?)).to eq(comment)
     end
 
-    it "can use headless policy" do
-      expect(Pundit.authorize(user, :publication, :create?)).to be_truthy
-    end
-
     it "returns the policy name symbol when passed record with headless policy" do
       expect(Pundit.authorize(user, :publication, :create?)).to eq(:publication)
     end
 
-    it "can use without a particular instance" do
-      expect(Pundit.authorize(user, Post, :show?)).to be_truthy
-    end
```

Comment:
Ah I see - I hadn't read this correctly before. Yes, it's good to delete these specs, since they're entirely covered by the `it "returns..."` specs.